var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -46,7 +47,7 @@ class Draco(CMakePackage):
     variant('parmetis', default=True,  description='Enable Parmetis support')
     variant('pythontools', default=False,  description='Enable support for extra python tools')
     variant('qt',       default=False, description='Enable Qt support')
-    variant('superlu_dist', default=True, description='Enable SuperLU-DIST support')
+    variant('superlu-dist', default=True, description='Enable SuperLU-DIST support')
```

Comment:
Can we keep this lowercase? In the spec syntax both the `~` and the `-` can be used to express the negation of a Boolean variant, so having `super-dist` can make things ambiguous.

Reply:
I want this variant to match the `trilinos` package because these dependencies are always synchronized in our build.  That is, it is confusing (error prone) to build `draco+superlu_dist` and `trilinos+superlu-dist`.  I am happy to make this an underscore if it can be done universally in spack.  Do you want me to create a PR with a universal change?

Reply:
Apologies for the lack of reply, this PR got buried under a never ending stream of other notifications. I'd be for using underscore everywhere. @sethrj Would it be fine with you to change that in `trilinos`?

Reply:
Yes I'm more than fine with that: there are a bunch of variants in spack that are inconsistent and our review process doesn't often catch the changes. I'll likely have to update the variant names in e4s as well.

Reply:
See https://github.com/spack/spack/pull/25841